locator/load_sketch.hh

```diff
@@ -223,7 +223,7 @@ public:
     // Returns 1 when shards are imbalanced, but it's not possible to balance them.
     load_type get_shard_imbalance(host_id node) const {
         auto minmax = get_shard_minmax(node);
-        return minmax.max() - minmax.max();
+        return minmax.max() - minmax.min();
```

Comment:
Should be a separate commit. Let's not squash independent changes.

Reply:
I could move it later, not earlier since it breaks the balancer random test.